go/ethadapter/mgmtcontractlib/mgmt_contract_ABI.go

```diff
@@ -3,11 +3,14 @@ package mgmtcontractlib
 import "github.com/ten-protocol/go-ten/contracts/generated/ManagementContract"
 
 const (
-	AddRollupMethod        = "AddRollup"
-	RespondSecretMethod    = "RespondNetworkSecret"
-	RequestSecretMethod    = "RequestNetworkSecret"
-	InitializeSecretMethod = "InitializeNetworkSecret" //#nosec
-	GetHostAddressesMethod = "GetHostAddresses"
+	AddRollupMethod                = "AddRollup"
+	RespondSecretMethod            = "RespondNetworkSecret"
+	RequestSecretMethod            = "RequestNetworkSecret"
+	InitializeSecretMethod         = "InitializeNetworkSecret" //#nosec
+	GetHostAddressesMethod         = "GetHostAddresses"
+	GetImportantContractKeysMethod = "GetImportantContractKeys"
+	SetImportantContractsMethod    = "SetImportantContractAddress"
+	GetImportantAddressMethod      = "importantContractAddresses"
```

Comment:
is this lower case intentional?

Reply:
Yeah, that's the method that solidity creates magically for the public field